tools/challenge-helper-scripts/helpers/new-challenge-prompts.ts

```diff
@@ -0,0 +1,45 @@
+import { prompt } from 'inquirer';
+import { challengeTypes } from '../../../client/utils/challenge-types';
+
+export const newChallengePrompts = async (): Promise<{
+  name: string;
+  title: string;
+  challengeType: string;
+}> => {
+  const title = await prompt({
```

Comment:
```suggestion
  const dashedName = await prompt({
```
'name' and 'title' are back to front, also name is a bit too generic imo

Reply:
Also, `prompt` is generic, so you can do something like 

```ts
  const { dashedName } = await prompt<{ dashedName: string }>({
    name: 'dashedName',
```
which is a little less awkward than either using `String` or `as { value: string }` to mollify TS.